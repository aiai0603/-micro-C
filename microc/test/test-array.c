Void main(Int n) {
  
    Int a[6];
    a[3] = 2;
    Int bb;
    bb=1;

    print("%d",a[3]);

}

