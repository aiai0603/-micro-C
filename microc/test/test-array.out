24 19 1 5 25 15 6 14 0 5 2 13 0 7 1 11 0 1222 12 15 -1 13 0 7 1 11 0 3 1 0 2 12 13 0 8 1 0 1 12 15 -1 13 0 7 1 11 22 15 -1 0 0 21 9